### PR TITLE
ref: replaced `dvc run` references with `dvc stage` and `dvc exp run` in `dvc dag`

### LINKED ISSUE #3183: ref: update `dvc run` references in `dvc dag`

### PR DESCRIPTION
**Pull request by iesahin**
replaced `dvc run` references with `dvc stage` and `dvc exp run`

Closes #3183

### CODE REVIEW

**Comment by dberenbaum** [comment]
😅 I should have seen this before I submitted #3223

**Comment by iesahin** [comment]
I can close this or merge and rebase the other. WDYT @jorgeorpinel ?

**Comment by jorgeorpinel** [comment]
If #2997 is closed this no longer seems needed (already addressed)...

File: content/docs/command-reference/dag.md [hunk 1]
```diff
@@ -21,19 +21,19 @@ show the full project DAG.
 ### Directed acyclic graph
 
 A data pipeline, in general, is a series of data processing
-[stages](/doc/command-reference/run) (for example, console commands that take an
-input and produce an outcome). The connections between stages are formed by the
-<abbr>output</abbr> of one turning into the <abbr>dependency</abbr> of another.
-A pipeline may produce intermediate data, and has a final result.
+[stages](/doc/command-reference/stage) (for example, console commands that take
```
Comment (jorgeorpinel) [inline content/docs/command-reference/dag.md:24]: OK I see we make more run-related changes here than in https://github.com/iterative/dvc.org/pull/3223/files#diff-7b6ee3d8c0ebb9e2a9dcf9077d16fda46133fb069f3879a3c38b0653fe00e850 but updating these links seems like a separate task. Let's create an issue to decide where should the "stage" definition be (probably a guide?) and then update links throughout.

**Comment by iesahin** [comment]
Closing this, as we merged #3223

File: content/docs/command-reference/dag.md (hunk 1 above)
Comment (iesahin) [reply content/docs/command-reference/dag.md:24]: Fine. Closed.

File: content/docs/command-reference/dag.md [hunk 2]
```diff
@@ -21,19 +21,19 @@ show the full project DAG.
...
 Data science and machine learning pipelines typically start with large raw
 datasets, include intermediate featurization and training stages, and produce a
 final model, as well as accuracy [metrics](/doc/command-reference/metrics).
 
 In DVC, pipeline stages and commands, their data I/O, interdependencies, and
 results (intermediate or final) are specified in `dvc.yaml`, which can be
-written manually or built using the helper command `dvc run`. This allows DVC to
-restore one or more pipelines later (see `dvc repro`).
+written manually or built using the helper command `dvc stage add`. This allows
+DVC to restore one or more pipelines later (see `dvc exp run` and `dvc repro`).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/dag.md:36]: Hmmmmm not sure we need to insert the `exp` reference here 🤔

Comment (jorgeorpinel) [reply content/docs/command-reference/dag.md:36]: I guess it doesn't hurt. OK I'll solve the conflicts and merge this instead ⌛

**Comment by jorgeorpinel** [comment]
Oh haha sorry, we're working on this synchronously it seems 😬

File: content/docs/command-reference/dag.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/dag.md:36]: p.s. we should probably check that we're mentioning `dvc dag` from `exp` references too?

Comment (jorgeorpinel) [reply content/docs/command-reference/dag.md:36]: OK. Extracted this into https://github.com/iterative/dvc.org/issues/3230 for now.

**Pull request closed without merging**